tests/test_lightningd.py

```diff
@@ -2335,6 +2337,7 @@ def test_permfail_htlc_out(self):
         bitcoind.generate_block(1)
         wait_forget_channels(l2)
 
+    @unittest.skipIf(not DEVELOPER, "needs DEVELOPER=1")
```

Comment:
Why does this need `DEVELOPER=1` now?  I do not see the test now using a `dev` command or option, unless I missed it?

(actually, for these `skipIf(not DEVELOPER` deals, should we not give the reason why we need `DEVELOPER=1` instead of the obvious `"needs DEVELOPER=1"`? E.g. `"needs --dev-foo-option"` or `"needs dev-foo-command so we can foo the funds before barring them"`)

Reply:
Yeah, this is a bit hidden, but we set the `--dev-broadcast-interval` if we have `DEVELOPER=1` which speeds up the staggered broadcast needed in this test 30x, without it we'd just be sitting there for over a minute and wait for the gossip to sync. I changed the `needs DEVELOPER=1` to hint at this requirement.